src/main/java/uk/gov/hmcts/reform/ccd/documentam/dto/DocumentUploadRequest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.ccd.documentam.dto;
+
+import io.swagger.annotations.ApiParam;
+import lombok.Data;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.ccd.documentam.dto.validation.EnumValue;
+import uk.gov.hmcts.reform.ccd.documentam.model.enums.Classification;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants.INPUT_STRING_PATTERN;
+
+@Data
+public class DocumentUploadRequest {
+    @ApiParam(value = "List of file to be uploaded", required = true)
+    @NotNull(message = "Provide some file to be uploaded")
+    @Size(min = 1, message = "Please provide at least one file to be uploaded.")
+    private final List<MultipartFile> files;
+
+    @ApiParam(value = "Security classification for the file", required = true)
+    @NotNull(message = "Please provide Classification")
+    @EnumValue(enumClass = Classification.class)
+    private final String classification;
```

Comment:
Can't Spring takes care of this automatically? I mean like `private Classification classification;`

Reply:
Maybe but doing that would mean losing the ability to test the endpoint at unit test level with invalid values.   There's probably a better way but I think this would suffice